docs/examples/widgets/progress_bar_isolated.py

```diff
@@ -0,0 +1,26 @@
+from textual.app import App, ComposeResult
+from textual.containers import Center, Middle
+from textual.widgets import ProgressBar
+
+
+class IndeterminateProgressBar(App[None]):
+    def compose(self) -> ComposeResult:
+        with Center():
+            with Middle():
+                yield ProgressBar()
+
+    def key_f(self) -> None:
+        # Freeze time for the indeterminate progress bar.
+        self.query_one(ProgressBar).query_one("#bar")._elapsed_time = lambda: 5
+
+    def key_t(self) -> None:
+        # Freeze time to show always the same ETA.
+        self.query_one(ProgressBar).query_one("#eta")._elapsed_time = lambda: 4.2
+        self.query_one(ProgressBar).update(total=100, progress=39)
```

Comment:
We need this so that the screenshots in the tests and in the docs always look the same.
I don't like this assignment to `_elapsed_time` to be so publicly exposed but I couldn't think of anything better.

Reply:
In a test, this would be fine. But I'm afraid that we can't do this in an example. Call me if you want to brainstorm an alternative solution.